### PR TITLE
fix api/version.bats

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux <victorvieux@gmail.com>

### CODE REVIEW

**Comment by vieux** [comment]
ping @aluzzardi

File: test/integration/api/version.bats [hunk 1]
```diff
@@ -12,14 +12,14 @@ function teardown() {
 	start_docker 1
 	swarm_manage
 
-	# version
-	run docker_swarm version
+	# Get version output
+	out=$(docker_swarm version)
+
+	run bash -c "echo '$out' | grep swarm/"
```
Comment (aluzzardi) [inline test/integration/api/version.bats]: I think this test was originally overkill. Just run docker_swarm version and check that `$output =~ swarm/`

Comment (vieux) [inline test/integration/api/version.bats]: ok

**Comment by vieux** [comment]
@aluzzardi PTAL

**Pull request merged**